python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/mpls.schema.yml

```
@@ -33,3 +33,260 @@ keys:
+                      max: 4294967295
+                    password_type:
+                      description: Configure authentication password type.
+                      type: str
+                      required: true
+                      convert_types:
+                        - int
+                      valid_values:
+                        - '0'
+                        - '7'
+                        - '8a'
+                    password:
+                      type: str
+                      required: true
+                      description: Password string.
+              active_index:
+                description: Use index as active password.
+                type: int
+                convert_types:
+                  - str
+              sequence_number_window:
+                description: |-
+                  Index in the sequence.
+                  Size of reorder window.
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 255
+              type:
+                description: Authentication mechanism.
+                type: str
+                valid_values:
+                  - 'md5'
+                  - 'none'
+          neighbors:
+            description: Neighbor-specific configuration.
+            type: list
+            primary_key: ip
+            items:
+              type: dict
+              keys:
+                ip:
+                  description: Neighbor's interface IPv4 or IPv6 address.
+                  type: str
+                authentication:
+                  description: Configure cryptographic authentication.
+                  type: dict
+                  required: true
+                  keys:
+                    index:
+                      description: Password index.
+                      type: int
+                      min: 1
+                      max: 4294967295
+                    type:
+                      description: Authentication mechanism.
+                      type: str
+                      valid_values:
+                        - 'md5'
+                        - 'none'
+          ip_access_group:
+            description: |-
+              IP config commands.
+              Access list name.
+            type: str
+          ipv6_access_group:
+            description: |-
+              IPv6 config commands.
+              Access list name.
+            type: str
+          fast_reroute:
+            description: Configure fast reroute.
+            type: dict
+            keys:
+              mode:
+                description: |-
+                  Fast reroute mode.
+                  Protect against failure of the next link.
+                  Protect against failure of the next node.
+                  Disable fast reroute.
+                type: str
+                valid_values:
+                  - 'link-protection'
+                  - 'node-protection'
+                  - 'none'
```

```suggestion
                  - link-protection
                  - node-protection
                  - none
```
Same changes at all other places